src/main/java/teammates/client/remoteapi/RemoteApiClient.java

```diff
@@ -8,21 +8,31 @@
 import teammates.test.driver.TestProperties;
 
 public abstract class RemoteApiClient {
-
+    private final String LOCALHOST = "localhost";
+    
     protected void doOperationRemotely() throws IOException {
         TestProperties testProperties = TestProperties.inst();
 
+        String appDomain = testProperties.TEAMMATES_REMOTEAPI_APP_DOMAIN;
+        int appPort = testProperties.TEAMMATES_REMOTEAPI_APP_PORT;
+        
         System.out.println("--- Starting remote operation ---");
-        System.out.println("Going to connect to:"
-                + testProperties.TEAMMATES_REMOTEAPI_APP_DOMAIN + ":"
-                + testProperties.TEAMMATES_REMOTEAPI_APP_PORT);
-
-        RemoteApiOptions options = new RemoteApiOptions().server(
-                testProperties.TEAMMATES_REMOTEAPI_APP_DOMAIN,
-                testProperties.TEAMMATES_REMOTEAPI_APP_PORT).credentials(
-                testProperties.TEST_ADMIN_ACCOUNT,
-                testProperties.TEST_ADMIN_PASSWORD);
-
+        System.out.println("Going to connect to:" + appDomain + ":" + appPort);
+
+        RemoteApiOptions options = new RemoteApiOptions().server(appDomain, appPort);
+
+        boolean isDevServer = appDomain.equals(LOCALHOST);
+        if (isDevServer) {
+            // Dev Server doesn't require credential.
+            options.useDevelopmentServerCredential();
+        } else {
+            // If you are trying to run script on Staging Server:
```

Comment:
But most of contributors/developers may not need to install Cloud SDK unless they have to write/run remote scripts. So I doubt it's worth to be considered as a step in setting up Developer Environment. The reason I put the instruction there because this is where the exception will be thrown when we try to send request with credential.